lib/android-helpers.js

```diff
@@ -89,43 +89,29 @@ helpers.ensureNetworkSpeed = function (adb, networkSpeed) {
 };
 
 helpers.ensureDeviceLocale = async function (adb, language, country) {
-  let haveLanguage = language && typeof language === "string";
-  let haveCountry = country && typeof country === "string";
-  if (!haveLanguage && !haveCountry) {
+  let hasLanguage = language && _.isString(language);
```

Comment:
The presence check is not strictly speaking necessary here. `_.isString(null)` and `_.isString(undefined)` both return `false`, so `language && _.isString(language)` will always return the same as `_.isString(language)`

Reply:
But more importantly, the variables are no longer, in this implementation, used beyond the error check, so they are not necessary at all. The following would suffice
```js
if (!_.isString(language) && !_.isString(country)) {
  logger.warn(...);
  return;
}
// ...
```